newIDE/app/src/InstancesEditor/SelectedInstances.js

```diff
@@ -176,5 +207,12 @@ export default class InstancesSelection {
       bottomResizeButtonPos,
       smallButtonSize
     );
+    this._renderButton(
+      show,
+      this.rotateButton,
+      rotateButtonPos,
+      smallButtonSize,
+      1
```

Comment:
Use CIRCLE_BUTTON_SHAPE instead of 1 here

Reply:
(that's the point of making CIRCLE_BUTTON_SHAPE/RECTANGLE_BUTTON_SHAPE ;)

Reply:
oops sorry, I thought I have changed it in a previous request.
Fixed now